### ISSUE TITLE
Form field missing a label

### ISSUE THREAD

**Issue by pooriamehregan** [Informasjonsforvaltning/fdk-issue-tracker#678]
[Form field missing a label - Issues - Accessibility NextGen - Accessibility.pdf](https://github.com/Informasjonsforvaltning/fdk-issue-tracker/files/10402285/Form.field.missing.a.label.-.Issues.-.Accessibility.NextGen.-.Accessibility.pdf)

**Comment by pooriamehregan** [comment]
Lenke under ser ut til å være falsk positive:
- https://data.norge.no/public-services-and-events?eventType=https%3A%2F%2Fdata.norge.no%2Fconcepts%2F298%2Chttps%3A%2F%2Fdata.norge.no%2Fconcepts%2F311%2Chttps%3A%2F%2Fdata.norge.no%2Fconcepts%2F332%2Chttps%3A%2F%2Fdata.norge.no%2Fconcepts%2F312
- https://data.norge.no/public-services-and-events?eventType=https%3A%2F%2Fdata.norge.no%2Fconcepts%2F298%2Chttps%3A%2F%2Fdata.norge.no%2Fconcepts%2F332%2Chttps%3A%2F%2Fdata.norge.no%2Fconcepts%2F311%2Chttps%3A%2F%2Fdata.norge.no%2Fconcepts%2F312
- https://data.norge.no/public-services-and-events?eventType=https%3A%2F%2Fdata.norge.no%2Fconcepts%2F311%2Chttps%3A%2F%2Fdata.norge.no%2Fconcepts%2F298%2Chttps%3A%2F%2Fdata.norge.no%2Fconcepts%2F332%2Chttps%3A%2F%2Fdata.norge.no%2Fconcepts%2F312
- https://data.norge.no/public-services-and-events?eventType=https%3A%2F%2Fdata.norge.no%2Fconcepts%2F311%2Chttps%3A%2F%2Fdata.norge.no%2Fconcepts%2F332%2Chttps%3A%2F%2Fdata.norge.no%2Fconcepts%2F298%2Chttps%3A%2F%2Fdata.norge.no%2Fconcepts%2F312
